modular_zubbers/code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/gun/ballistic/automatic/wt550
+	can_bayonet = FALSE
+	can_suppress = FALSE
+
+/obj/item/gun/ballistic/automatic/wt550/security
+	name = "\improper WT-551 Autorifle"
+	desc = "A heavier, bulkier automatic variant of the WT-550, and now with 99% less discombobulation! It's back, baby. Uses 4.6x30mm rounds. Recommended to hold with two hands."
```

Comment:
If you referenced that this is either a civilian export variant/  Sol Firearm Code Compliant (which would explain why this thing is still around versus the WT-550) I'd dig it